### ISSUE TITLE
Excluding the current day from the shortcut integer values

### ISSUE THREAD

**Issue by FilipQL** [chronotruck/vue-ctk-date-time-picker#210]
As stated in the documentation:

> If the value of shortcut is a number (Integer), this number correspond to number of day (for 5 --> Last 5 days)

But, the problem is that it will select **today** _+ 5 previous days_.  For example, this shortcut:
```
{ label: 'Last 7 days', value: 7, isHover: false, isSelected: false },
```
will select **8** days (_today_ & _previous 7 days_). 

In my opinion - it should select only _previous 7 days_, without _today_. 
But anyway, if I see well - currently there's no such value that will exclude _today_ from selection. 
It would be nice to have another type of value, for example something like: `7wt` which would mean: _Last **7** days **w**ithout **t**oday_.

**Comment by WilliamDASILVA** [comment]
Hello @FilipQL 
Thank you for taking the time to make this issues.
It would be a breaking change if we excluded the current day from the values; but introducing something new (like you did with the `7wt`- seems too much.

Keeping this issue open. I'll try to find a more sexy way to handle this.

**Comment by der-lars** [comment]
Yes, it would be a big change but on the other hand it's just wrong the way it is.

If I would ask you "what did you do the last 2 days?" you would probably tell me about yesterday and the day before. At least I would... :)
And filters like this in other tools work like that.

Anyways, good luck with finding a way we haven't thought of yet! 
If you don't find any, please think again about changing this universally.

**Comment by WilliamDASILVA** [comment]
Hello there,

As said in issue #232, as I can see between different context we cannot agree on the definition of "last x days" (does it includes today or not).
For that, a workaround would be to use custom range value through a function.

E.g:

```js
[
  {
    key: 'customValue',
    label: 'My custom thing',
    value: () => {
      return {
        start: moment(),
        end: moment().add(2, 'days')
      }
    }
  },
];
```

Hope that helps to fix the issue temporarily.

**Comment by FilipQL** [comment]
@WilliamDASILVA Thanks! :+1:

**Comment by FilipQL** [comment]
Hey @WilliamDASILVA,
I see that the documentation ("README.md") includes this new feature, but if I see well - it's still not included in the latest release (currently `v2.3.1`). I guess that it will be released on Monday among with this: https://github.com/chronotruck/vue-ctk-date-time-picker/issues/218 (`v2.4.0`)?

**Comment by WilliamDASILVA** [comment]
Hello @FilipQL,
Indeed the changes are not released yet. I'm releasing the `v2.4.0` this afternoon!
